source/crud/tests/unified/bulkWrite-replaceOne-sort.yml

```
@@ -0,0 +1,94 @@
+description: BulkWrite replaceOne-sort
+
+schemaVersion: "1.0"
+
+createEntities:
+  - client:
+      id: client0
+      observeEvents: [ commandStartedEvent, commandSucceededEvent ]
+  - database:
+      id: database0
+      client: client0
+      databaseName: crud-tests
+  - collection:
+      id: collection0
+      database: database0
+      collectionName: coll0
+
+initialData:
+  - collectionName: coll0
+    databaseName: crud-tests
+    documents:
+      - { _id: 1, x: 11 }
+      - { _id: 2, x: 22 }
+      - { _id: 3, x: 33 }
+
+tests:
+  - description: BulkWrite replaceOne with sort option
+    runOnRequirements:
+      - minServerVersion: "8.0"
+    operations:
+      - object: collection0
+        name: bulkWrite
+        arguments:
+          requests:
+            - replaceOne:
+                filter: { _id: {$gt: 1 } }
```

```suggestion
                filter: { _id: { $gt: 1 } }
```

Consistent whitespace.